tests/integration/targets/zypper/tasks/zypper.yml

```diff
@@ -442,3 +442,23 @@
     name: gnu-netcat
     state: present
     force_resolution: True
+
+- name: remove gnu-netcat and re-install netcat-openbsd
+  zypper:
+    name:
+      - -gnu-netcat
+      - +netcat-openbsd
+    state: present
+
+- name: Install gnu-netcat with force_resolution and replacefiles
+  zypper:
+    name: gnu-netcat
+    state: present
+    force_resolution: true
+    replacefiles: true
+
+- name: Dist upgrade with allow_vendor_change
+  zypper:
+    name: '*'
```

Comment:
Please limit this to something: a) this test now downlodas *a lot* of packages, and b) it upgrades the Python 2 installation, thus modifying the environment.

Reply:
The dist-upgrade state mandate '*' as name.. So I just removed it for now..

Reply:
Ah, too bad. In that case, not testing it for now is probably best...